src/tv.c

```diff
@@ -5334,7 +5334,8 @@ static void DoTVShow3CheersForPokeblocks(void)
         if (show->threeCheers.sheen > 24)
         {
             StringCopy(gStringVar2, gText_Excellent);
-        } else if (show->threeCheers.sheen > 22)
+        }
+        else if (show->threeCheers.sheen > 22)
```

Comment:
As above, for pret or another PR

Reply:
Incorporated pret merge that fixes this.